Geometry/MTDCommonData/interface/MTDBaseNumber.h

```diff
@@ -19,20 +19,20 @@ class MTDBaseNumber {
   ~MTDBaseNumber() {}
 
   void setSize(const int& size);
-  void addLevel(const std::string& name, const int& copyNumber);
+  void addLevel(const std::string_view& name, const int& copyNumber);
```

Comment:
The `std::string_view`s are generally recommended to be passed around by value.

Reply:
thanks for the comment, updated